lib/sass_spec/directory.rb

```diff
@@ -195,6 +200,13 @@ def to_s
 
   private
 
+  # Returns whether `file` contains enough `../` references to reach outside
+  # this directory.
+  def _reaches_out?(file)
+    depth = file.path.count("/")
```

Comment:
Do we know if file paths have trailing slashes?

Reply:
The HRX spec guarantees that they won't; otherwise they'd be parsed as `HRX::Directory`s instead.